importer/src/test/java/org/hiero/mirror/importer/parser/domain/RecordFileBuilder.java

```diff
@@ -197,6 +219,24 @@ private Supplier<RecordItemBuilder.Builder<?>> buildTemplate() {
             if (subType != SubType.STANDARD) {
                 return switch (subType) {
                     case TOKEN_TRANSFER -> () -> recordItemBuilder.cryptoTransfer(TransferType.TOKEN);
+                    case CONTRACT_CALL -> {
+                        var transactionID = TransactionID.newBuilder()
+                                .setNonce(nonce)
+                                .setAccountID(AccountID.newBuilder()
+                                        .setAccountNum(4100)
+                                        .build())
```

Comment:
Shouldn't hardcode shard, realm, and num.

```suggestion
                                .setAccountID(recordItemBuilder.accountId())
```